### PR TITLE
TAJO-1487: Kafka Scanner for kafka strage.

### PR DESCRIPTION
**Pull request by combineads**
Have to implement Scanner interface for Kafka storage.
The scan by split into many fragments. Scanner and Fragment are 1 to 1 mapping.
For this, need to feature like this,
Create Fragment class
Implement 'Tuple next()' method in Scanner.
etc..
The issue is related to TAJO-1480, TAJO-1502. So first, must be commit TAJO-1480 and TAJO-1502.

### CODE REVIEW

File: tajo-storage/tajo-storage-kafka/src/main/java/org/apache/tajo/storage/kafka/KafkaFragment.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  public void setStartKey(int partitionId, long startOffset) {
+    this.startKey = new KafkaFragmentKey(partitionId, startOffset);
+  }
+
+  public void setEndKey(int partitionId, long lastOffset) {
+    this.endKey = new KafkaFragmentKey(partitionId, lastOffset);
+  }
+
+  public static class KafkaFragmentKey implements Comparable<KafkaFragmentKey> {
+    private final Integer partitionId;
```
Comment (jinossy) [inline tajo-storage/tajo-storage-kafka/src/main/java/org/apache/tajo/storage/kafka/KafkaFragment.java]: partitionId and offset should be primitive

File: tajo-storage/tajo-storage-kafka/src/main/java/org/apache/tajo/storage/kafka/KafkaScanner.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+  private List<ConsumerRecord<byte[], byte[]>> records = null;
+
+  private Schema schema;
+  private TableMeta meta;
+  private TableStats tableStats;
+  private KafkaFragment fragment;
+  private Column[] targets;
+  private TextLineDeserializer deserializer;
+  private AtomicBoolean finished = new AtomicBoolean(false);
```
Comment (jinossy) [inline tajo-storage/tajo-storage-kafka/src/main/java/org/apache/tajo/storage/kafka/KafkaScanner.java]: There is no race condition in scanner.

**Comment by jinossy** [comment]
rebase please

**Comment by combineads** [comment]
@jinossy Done rebase.
Thanks for your reviews.

**Comment by jinossy** [comment]
+1 LGTM!!

**Pull request closed without merging**